README.md

```diff
@@ -30,105 +48,6 @@ Get more information about this tool in the [contributing][] document.
 [report]: https://hyper.rs/h3/ci/compliance/report.html#/
 [contributing]: CONTRIBUTING.md
 
-## Features
-
-* HTTP/3 client and server implementation
-* Async only API
-* QUIC transport abstraction via traits in the [`quic`](./h3/src/quic.rs) module
-* Supported QUIC implementations to date are
-  [Quinn](https://github.com/quinn-rs/quinn) ([h3-quinn](./h3-quinn/))
-  and [s2n-quic](https://github.com/aws/s2n-quic)
-  ([s2n-quic-h3](https://github.com/aws/s2n-quic/tree/main/quic/s2n-quic-h3))
-
-## Overview
-
-* **h3** HTTP/3 implementation
-* **h3-quinn** QUIC transport implementation based on [Quinn](https://github.com/quinn-rs/quinn/)
```

Comment:
Why did you remove the Overview?

Reply:
It's kinda merged into the intro part. Is there anything missing from the intro part? I did the cleanup coz it seems like people just jammed stuff into the README and it wasn't really maintained. The structure was kind messy. I structured it a little: overview, status, getting started, interop info

Reply:
This is the file structure, folder `h3` is the actual http/3 implementation. An in `h3-quinn` there is the quinn stuff.

I think it\'s good because it gives a quick overview of how the project is structured.